<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Robert Grimes</author>
 <class>Monitor</class>
 <widget class="QWidget" name="Monitor">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Label Printer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>printer.png</normaloff>printer.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:white</string>
  </property>
  <widget class="QFrame" name="mainFrame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>800</width>
     <height>480</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>480</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton:focus {
	border: none;
	outline: none;
}
QPushButton{
background: transparent;
 border-radius: 10px;
  background-image: linear-gradient(#335b71 45%, #03324c 55%);


}
QComboBox QAbstractItemView {
  border: 1px solid grey;
  background: white;
  selection-background-color: blue;
}
QComboBox {
  background: white;
}
</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="mainPage">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QLabel" name="bottleLbl_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>134</width>
        <height>351</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>syringe.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QProgressBar" name="printProgressBar">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>390</y>
        <width>321</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Kartika</family>
       </font>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QPushButton" name="setupBtn">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>380</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Kartika</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>images/shutdown.png</normaloff>
        <normalon>setup.png</normalon>images/shutdown.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>105</width>
        <height>45</height>
       </size>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>621</width>
        <height>191</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="expirationLbl">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>100</y>
         <width>291</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Kartika</family>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:red;background:transparent;</string>
       </property>
       <property name="text">
        <string>2/8/2021 11:35 AM</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="currentDateLbl">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>261</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Kartika</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(44,153,242);background:transparent;</string>
       </property>
       <property name="text">
        <string>Thursday, November 22nd</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="currentTimeColonLbl">
       <property name="geometry">
        <rect>
         <x>128</x>
         <y>26</y>
         <width>21</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Kartika</family>
         <pointsize>56</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(44,153,242);background:transparent;</string>
       </property>
       <property name="text">
        <string>:</string>
       </property>
      </widget>
      <widget class="QLabel" name="currentTimeHHLbl">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>30</y>
         <width>99</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Kartika</family>
         <pointsize>56</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(44,153,242);background:transparent;</string>
       </property>
       <property name="text">
        <string>10</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>50</y>
         <width>241</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Kartika</family>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(44,153,242);background:transparent;</string>
       </property>
       <property name="text">
        <string>Expiration On</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="currentTimeMMLbl">
       <property name="geometry">
        <rect>
         <x>152</x>
         <y>30</y>
         <width>105</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Kartika</family>
         <pointsize>56</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(44,153,242);background:transparent;</string>
       </property>
       <property name="text">
        <string>22</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>781</width>
        <height>461</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:4px solid blue; border-radius: 25px;</string>
      </property>
      <widget class="QPushButton" name="printBtn">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>240</y>
         <width>181</width>
         <height>91</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Kartika</family>
         <pointsize>36</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border:0px solid red</string>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>print.png</normaloff>
         <normalon>print-down.png</normalon>
         <selectedoff>print.png</selectedoff>
         <selectedon>print-down.png</selectedon>print.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>150</width>
         <height>82</height>
        </size>
       </property>
      </widget>
      <widget class="QLabel" name="lableLbl">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>230</y>
         <width>171</width>
         <height>100</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:2px solid red</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="thisMedicationLbl">
       <property name="geometry">
        <rect>
         <x>497</x>
         <y>250</y>
         <width>161</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border:0px solid red</string>
       </property>
       <property name="text">
        <string>This Medication</string>
       </property>
      </widget>
      <widget class="QLabel" name="expirationTimeLbl">
       <property name="geometry">
        <rect>
         <x>497</x>
         <y>290</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border:0px solid red</string>
       </property>
       <property name="text">
        <string>at</string>
       </property>
      </widget>
      <widget class="QLabel" name="expirationDateLbl">
       <property name="geometry">
        <rect>
         <x>497</x>
         <y>270</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border:0px solid red</string>
       </property>
       <property name="text">
        <string>Expires</string>
       </property>
      </widget>
     </widget>
     <zorder>widget_2</zorder>
     <zorder>bottleLbl_2</zorder>
     <zorder>printProgressBar</zorder>
     <zorder>setupBtn</zorder>
     <zorder>groupBox</zorder>
    </widget>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="saveSetupBtn">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>220</y>
        <width>191</width>
        <height>82</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Kartika</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normalon>change.png</normalon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>150</width>
        <height>82</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="shutdownBtn">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>380</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Kartika</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="icon">
       <iconset>
        <normalon>shutdown.png</normalon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>105</width>
        <height>45</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="goBackPage1Btn">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>40</y>
        <width>191</width>
        <height>82</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Kartika</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normalon>goback.png</normalon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>150</width>
        <height>82</height>
       </size>
      </property>
     </widget>
     <widget class="QLineEdit" name="dateTxt">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Kartika</family>
        <pointsize>16</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>24</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Kartika</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Date:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>74</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Kartika</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Time:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="timeTxt">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>70</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Kartika</family>
        <pointsize>16</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="changeTimeBtn">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>70</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Kartika</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="icon">
       <iconset>
        <normalon>change-arrow.png</normalon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>105</width>
        <height>45</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="changeDateBtn">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>20</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Kartika</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="icon">
       <iconset>
        <normalon>change-arrow.png</normalon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>105</width>
        <height>45</height>
       </size>
      </property>
     </widget>
     <widget class="QLineEdit" name="expirationTxt">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>120</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Kartika</family>
        <pointsize>16</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>120</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Kartika</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Hours:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="expirationHoursBtn">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>120</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Kartika</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="icon">
       <iconset>
        <normalon>change-arrow.png</normalon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>105</width>
        <height>45</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="versionLbl">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>430</y>
        <width>281</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Kartika</family>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="textFormat">
       <enum>Qt::RichText</enum>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>781</width>
        <height>461</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:4px solid blue; border-radius: 25px;</string>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>430</y>
         <width>121</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Kartika</family>
         <pointsize>8</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border:0px solid red</string>
       </property>
       <property name="text">
        <string>&amp;copy; 2021,etherpost.com</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <zorder>widget</zorder>
     <zorder>saveSetupBtn</zorder>
     <zorder>shutdownBtn</zorder>
     <zorder>goBackPage1Btn</zorder>
     <zorder>dateTxt</zorder>
     <zorder>label</zorder>
     <zorder>label_2</zorder>
     <zorder>timeTxt</zorder>
     <zorder>changeTimeBtn</zorder>
     <zorder>changeDateBtn</zorder>
     <zorder>expirationTxt</zorder>
     <zorder>label_3</zorder>
     <zorder>expirationHoursBtn</zorder>
     <zorder>versionLbl</zorder>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
